packages/oaf/src/commands/clan/skills.ts

```diff
@@ -161,32 +170,47 @@ export async function execute(interaction: ChatInputCommandInteraction) {
       await getParticipationFromCurrentRaid(),
     );
 
-    const skillsOwed = [...participation.entries()]
-      .filter(([playerId]) => players.get(playerId)?.doneWithSkills !== true)
-      .map(
-        ([playerId, { skills, kills }]) =>
-          [playerId, Math.floor((kills + 450) / 900) - skills] as const,
-      )
-      .filter(([, owed]) => owed > 0)
-      .sort(([, a], [, b]) => b - a)
-      .map(
-        ([playerId, owed]) =>
-          `${formatPlayer(players.get(playerId), playerId)}: ${pluralize(
-            owed,
-            "skill",
-          )}.`,
-      );
-
-    await interaction.editReply({
-      content: null,
-      embeds: [
-        {
-          title: "Skills owed",
-          fields: columnsByMaxLength(skillsOwed),
-        },
-      ],
-      allowedMentions: { users: [] },
-    });
+    if (input) {
+      const identifyResult = await identifyPlayer(input);
+      if (typeof identifyResult === "string") {
+        return void (await interaction.editReply(identifyResult));
+      }
+      const [, knownPlayer] = identifyResult;
+      if (!knownPlayer)
+        return void (await interaction.editReply(
+          `Player matching ${input} could not be found.`,
+        ));
+      return void (await interaction.editReply(
+        `${knownPlayer.playerName} is currently ${knownPlayer.doneWithSkills ? "" : "not yet "} done with skills. They have performed ${knownPlayer.kills} kills and received ${knownPlayer.skills} skills in ASS Dread instances.`,
```

Comment:
Change this so there's no double space. Fix the plurals in the case of kills or skills === 1